mage_ai/data_preparation/templates/data_exporters/google_sheets.py

```diff
@@ -33,4 +33,4 @@ def export_to_google_sheet(df: DataFrame, **kwargs) -> None:
     GoogleSheets.with_config(ConfigFileLoader(config_path, config_profile)).export(
         df, 
         sheet_url=sheet_url
-    )
+    )
```

Comment:
did you mean to make this change? we usually keep an empty line at the end of a file